io.github.jakubmelka.pdf4qt.json

```
@@ -0,0 +1,52 @@
+{
+    "id": "io.github.jakubmelka.pdf4qt",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland"
+    ],
```

```suggestion
        "--share=ipc",
        "--socket=fallback-x11",
        "--socket=wayland"
    ],
```

This is needed. Also if wayland is supported, x11 should just be fallback.